lib/rust/ensogl/component/text/src/font/embedded/Cargo.toml

```diff
@@ -13,7 +13,9 @@ enso-prelude = { path = "../../../../../../prelude" }
 ensogl-text-font-family = { path = "../../font/family" }
 
 [build-dependencies]
+enso-build = { git = "https://github.com/enso-org/ci-build", branch = "develop" }
```

Comment:
This is very bad. `ci-build` has HUGE pile of dependencies. It will make compiling this file VERY long. Let's have a call about it please.

Reply:
:heavy_check_mark: discussed